mu/integration_tests/test_mci.py

```diff
@@ -3,31 +3,34 @@
 
 from mu.integration_tests import int_test
 from mu.sims.mci import Mci
+from mu.protogen import stores_pb2
 
+MCP2515_KEY = (stores_pb2.MuStoreType.MCP2515, 0)
```

Comment:
Should this go with the MCP2515 sim? Same for `MCP23008_KEY` in test_mcp23008.py